### ISSUE TITLE
Shape of the input features

### ISSUE THREAD

**Issue by dangkh** [zeroQiaoba/GCNet#3]
Hi, I truly like what you did and I'm reproducing your work with another dataset. But, I have problems with the shape of the input features. 

At file ["gcnet/train_gcnet.py"](https://github.com/zeroQiaoba/GCNet/blob/main/gcnet/train_gcnet.py#L223), line 223, you mention the shape after reading in dataloader
`audio_host, text_host, visual_host: [seqlen, batch, dim]`

However, the shape of the data in the dataloader is usually `[batch, seqlen, dim]` and without any shape transpose in your code. Whether there are any piece of code missing from your work?

**Comment by zeroQiaoba** [comment]
Actually, based on pytorch=1.8.0, https://pytorch.org/docs/1.8.0/generated/torch.nn.LSTM.html?highlight=nn%20lstm#torch.nn.LSTM, torch.nn.LSTM takes  (seq_len, batch, input_size) as the input.
